catalog/fonts/Oswald/stylesheet.css

```diff
@@ -0,0 +1,54 @@
+@font-face {
```

Comment:
And this one — the font-face code should rather be moved to a corresponding "fonts" CSS file